### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by sinchang** [stitchesjs/stitches#776] [Ideas] [answered]
Is there any solution for adding RTL support?

**Comment by ekafyi** [comment]
You can use the standard CSS [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) property in the [globalCss](https://stitches.dev/docs/api#globalcss), example:

```js
const globalStyles = globalCss({
  body: { 
    direction: rtl;
    // other css
  },
});
```

**Comment by jonathantneal** [comment, marked as answer]
Stitches supports CSS Logical Properties, which work with LTR and RTL out of the box.

```js
const globalStyles = globalCss({
  p: { 
    marginInlineStart: '1em', // padding-left when LTR, padding-right when RTL
  },
})
```

[Further reading](https://css-tricks.com/css-logical-properties/)
